test/lib/utils-spec.js

```diff
@@ -148,4 +148,20 @@ describe('utils', () => {
       conflict.should.equal(false);
     });
   });
+  describe('Whitelabel', () => {
+    it('should get whitelabel from an API token', async () => {
```

Comment:
so when you move this into loaded config, I would test the new case where you don't have the white label info yet and this call has to be made.